cime/config/e3sm/config_grids.xml

```diff
@@ -2213,7 +2270,8 @@
     <domain name="oRRS15to5">
       <nx>5778136</nx>
       <ny>1</ny>
-      <file grid="ice|ocn">$DIN_LOC_ROOT/share/domains/domain.ocn.oRRS15to5.160207.nc</file>
+      <!--file grid="ice|ocn">$DIN_LOC_ROOT/share/domains/domain.ocn.oRRS15to5.160207.nc</file-->
```

Comment:
What was the reason to replace this file? Related: did you upload new file(s) to input repo?

Reply:
Actually, that's a good question. I was having some trouble during domain checks with the new grids and the new way we generate mapping files (using TempestRemap instead of ESMF), and this was one of the changes I made when trying to debug that, thinking that the domains needed to be consistent with the maps. But I think this one shouldn't matter now that I look at it, I think the ocean domain is just copied straight through without any mapping. I have a test building now to see if I can get rid of this change. Will be good to narrow that down. Thanks for catching!

Reply:
I just reverted to the old file. My test confirmed that this change was not needed, it was just done for consistency with the other new files that were needed. Reverting to minimize changes in this PR.